src/nb_type.cpp

```diff
@@ -550,6 +551,12 @@ PyObject *nb_type_new(const type_data *t) noexcept {
 
     #undef COPY_FIELD
 
+    tp->tp_members = nullptr;
```

Comment:
Very minor: the `nullptr` assignment isn't needed as the output of `PyType_GenericAlloc` is zero-initialized.